apps/meteor/client/views/room/providers/ComposerPopupProvider.tsx

```diff
@@ -64,14 +67,16 @@ const ComposerPopupProvider = ({ children, room }: ComposerPopupProviderProps) =
 					const filterRegex = filter && new RegExp(escapeRegExp(filter), 'i');
 					const items: ComposerBoxPopupUserProps[] = [];
 
-					const users = Array.from(usersFromRoomMessages.values())
-						.filter((u) => !!u.ts && (filterRegex ? u.username.match(filterRegex) || u.name?.match(filterRegex) : true))
+					const roomMessageUsers = queryClient.getQueryData<RoomMessageUser[]>(roomMessageUsersQueryKeys.all(rid, uid)) ?? [];
+
+					roomMessageUsers
 						.sort((a, b) => b.ts.getTime() - a.ts.getTime())
-						.slice(0, suggestionsCount ?? 5)
-						.map((u) => {
-							u.suggestion = true;
-							return u;
-						});
```

Comment:
Where did this map go?

Reply:
I think you didnt get the question, you are filtering/mapping/sorting, but you are not using the result